loopchain/configure_default.py

```diff
@@ -186,6 +186,7 @@ class ConsensusAlgorithm(IntEnum):
 # Some older clients have a process that treats tx, which is delayed by more than 30 minutes, as a failure.
 # The engine limits the timestamp of tx to a lower value.
 ALLOW_TIMESTAMP_BOUNDARY_SECOND_IN_BLOCK = 60 * 15
+ALLOW_TIMESTAMP_BUFFER_SECOND_IN_VERIFIER = int(0.3 * 1_000_000)  # 300ms
```

Comment:
I don't know why I feel the word "ALLOW" means boolean value. It's weird for me. 
How about `TIMESTAMP_BUFFER_IN_VERIFIER` or `BUFFER_TIMESTAMP_IN_VERIFIER` without "ALLOW"?